Remove horizontal scroll bar

minkyngkm opened the pull request:
## Done
- Replaced the `<pre>` and `<code>` element with `<div>` with styles
- Added `_row-cli.scss` file in static/sass for styles

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check if the code snippet's animation is working well and it's responsive according to screen size
## Issue / Card

Fixes #127 

## Screenshots

![image](https://user-images.githubusercontent.com/57550290/132309704-8c326996-0c9d-4e5a-afe2-3de5c0184f52.png)

webteam-app commented:
Demo starting at https://microstack-run-128.demos.haus

bethcollins92 commented on templates/index.html:
```diff
@@ -40,7 +40,7 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
         <h3 class="p-heading--4">Fully functional OpenStack in 2 commands and ~20 minutes.</h3>
       </div>
-      <pre class="p-heading--two"><code>$ sudo <span class="typer" id="main" data-words="snap install microstack --beta --devmode,microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></code></pre>
+      <div class="row-cli__commands" style="margin-bottom: 1rem;">$ sudo<span class="row-cli__commands-list typer" id="main" data-words="snap install microstack --beta --devmode,microstack init --auto --control" data-delay="50" data-deleteDelay="5000" data-colors="#111111"></span></div>
```
Could you not add the margin bottom style to the `row-cli__commands` class?

minkyngkm replied:
Fixed! :)

minkyngkm replied:
Fixed 👍

bethcollins92 reviewed (commented):
I checked on an Iphone X screen and there was an ellipses in the command: 

<img width="259" alt="Screenshot 2021-09-07 at 13 41 50" src="https://user-images.githubusercontent.com/58959073/132346578-f4627db2-288a-46ab-9756-460d9269cd8c.png">

But this ellipses also happens on juju. @anthonydillon do you know if this is expected behaviour or if this is a bug?

<img width="217" alt="Screenshot 2021-09-07 at 13 42 45" src="https://user-images.githubusercontent.com/58959073/132346553-919d0cb7-9d86-41ed-8345-994b9b8888c4.png">

anthonydillon commented:
I see the same on my phone. Difficult to know really. Hiding any part of the command is bad. Maybe the command should wrap in small screens. Unless we can set the font size based on the `vw`.

clagom commented:
+1, wrap the command line seems like a good option for small screens

bethcollins92 commented:
Thanks for the feedback @anthonydillon and @clagom, let me know when it's ready for a review again @minkyngkm (happy to pair if needed). I'll create an issue on juju.is too.

minkyngkm commented:
Thank you @bethcollins92, @anthonydillon and @clagom 🙏🏻 . I have updated it. Hopefully it looks better! - [demo](https://microstack-run-128.demos.haus/)

For your reference, I updated the `line-height` of the command line from `1rem` to `1.2rem` in case of the second line appearing on small screens. Please let me know if it's okay Beth :)

bethcollins92 reviewed (approved):
Looks great @minkyngkm - good job!
